{"name": "test_wpa2_personal_ssid_5g[setup_profiles0]", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "get_configuration = {'access_point': [{'ip': '10.28.3.100', 'jumphost': True, 'jumphost_tty': '/dev/ttyAP3', 'mode': 'wifi5', ...}], 'cont...hy4'], '2.4G-Station-Name': 'sta00', '5G-Radio': ['1.1.wiphy5'], '5G-Station-Name': 'sta10', ...}, 'name': 'lanforge'}}\nget_apnos = <class 'apnos.apnos.APNOS'>\nfixtures_ver = <fixtures_2x.Fixtures_2x object at 0x7f15eeea1550>\nadd_allure_environment_property = <function add_allure_environment_property.<locals>.maker at 0x7f15eed20af0>\n\n    @fixture(scope='session')\n    def add_env_properties(get_configuration, get_apnos, fixtures_ver, add_allure_environment_property: Callable) -> None:\n        add_allure_environment_property('Access-Point-Model', get_configuration[\"access_point\"][0][\"model\"])\n        add_allure_environment_property('Access-Point-Firmware-Version',\n>                                       fixtures_ver.get_ap_version(get_apnos, get_configuration)[0].split(\"\\n\")[1])\nE       IndexError: list index out of range\n\nconftest.py:643: IndexError"}, "description": "Client Connectivity wpa2_personal ssid 5G\n           pytest -m \"client_connectivity and bridge and general and wpa2_personal and fiveg\"\n        ", "parameters": [{"name": "setup_profiles", "value": "{'mode': 'BRIDGE', 'ssid_modes': {'open': [{'ssid_name': 'ssid_open_2g_br', 'appliedRadios': ['2G'], 'security_key': 'something'}, {'ssid_name': 'ssid_open_5g_br', 'appliedRadios': ['5G'], 'security_key': 'something'}], 'wpa': [{'ssid_name': 'ssid_wpa_2g_br', 'appliedRadios': ['2G'], 'security_key': 'something'}, {'ssid_name': 'ssid_wpa_5g_br', 'appliedRadios': ['5G'], 'security_key': 'something'}], 'wpa2_personal': [{'ssid_name': 'ssid_wpa2_2g_br', 'appliedRadios': ['2G'], 'security_key': 'something'}, {'ssid_name': 'ssid_wpa2_5g_br', 'appliedRadios': ['5G'], 'security_key': 'something'}]}, 'rf': {}, 'radius': False}"}], "start": 1630542375632, "stop": 1630542375632, "uuid": "dc9f9d04-b2d8-432f-8c13-fddfed35170e", "historyId": "92de83726af0ae8915affc1f4b2a758f", "testCaseId": "cd93f654f2eddf817bed39e62a0aff44", "fullName": "e2e.basic.validation_of_operating_modes.bridge_mode.client_connectivity.test_general_security_modes.TestBridgeModeConnectivitySuiteA#test_wpa2_personal_ssid_5g", "labels": [{"name": "subSuite", "value": "Bridge Mode Client Connectivity : Suite-A"}, {"name": "feature", "value": "BRIDGE MODE CLIENT CONNECTIVITY"}, {"name": "suite", "value": "sanity"}, {"name": "story", "value": "wpa2_personal 5 GHZ Band"}, {"name": "tag", "value": "wpa2_personal"}, {"name": "tag", "value": "sudo"}, {"name": "tag", "value": "bridge"}, {"name": "tag", "value": "fiveg"}, {"name": "tag", "value": "ucentral"}, {"name": "tag", "value": "sanity_light"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_profiles')"}, {"name": "tag", "value": "suiteA"}, {"name": "tag", "value": "client_connectivity"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "uc_sanity"}, {"name": "tag", "value": "general"}, {"name": "parentSuite", "value": "e2e.basic.validation_of_operating_modes.bridge_mode.client_connectivity"}, {"name": "host", "value": "testing-86-whrbp"}, {"name": "thread", "value": "6-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2e.basic.validation_of_operating_modes.bridge_mode.client_connectivity.test_general_security_modes"}]}